part-spec/common/componentID.json

```diff
@@ -106,11 +106,16 @@
                     "example": "2022-03-31",
                     "type": "string"
                 },
+                "eDataheetSpecRevision": {
+                    "description": "revision of the digital datasheet specifications used to create the digital datasheet",
+                    "example": "rev 1.0",
```

Comment:
suggest that we make this an enum (where we add a string each time we release) - that way we won't get revisions that we can't match against anything.

so "enum": ["1.0"]

Reply:
Changed